core/modules/macros/charcode.js

```diff
@@ -0,0 +1,37 @@
+/*\
+title: $:/core/modules/macros/charcode.js
+type: application/javascript
+module-type: macro
+
+Macro to return characters using the character-codes as input.
+If the code isn't found the character is returned.
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "charcode";
+
+exports.params = [
+	{name: "codes"},
+];
+
+exports.run = function(codes) {
+	var lookup = $tw.config.lookup,
+		tokens = codes.trim().split(/\s+/),
+		results = [];
+	tokens.map(function(name) {
```

Comment:
I know that some of them have crept in, but we don't currently use `.map` in the core.

Reply:
I'll change that